pep8.py

```diff
@@ -1328,7 +1328,12 @@ def _add_check(check, kind, codes, args):
                 codes = ERRORCODE_REGEX.findall(check.__doc__ or '')
             _add_check(check, args[0], codes, args)
     elif inspect.isclass(check):
-        if inspect.getargspec(check.__init__)[0][:2] == ['self', 'tree']:
+        if sys.version_info[0] >= 3:
```

Comment:
```py
if sys.version_info >= (3, 3):
```

That will suffice for what we need. `inspect.signature` was introduced in 3.3 and not before.